release-schema.json

```
@@ -104,6 +103,67 @@
         }
       },
       "minProperties": 1
+    },
+    "Prizes": {
+      "title": "Prizes",
+      "description": "Information about the prizes awarded in the design contest.",
+      "type": "object",
+      "properties": {
+        "description": {
+          "title": "Description",
+          "description": "The description of the prizes.",
```

Per https://ocds-standard-development-handbook.readthedocs.io/en/latest/meta/schema_style_guide.html#descriptions

```suggestion
          "description": "A description of the prizes. Structured information should be provided in `.details`.",
```